src/desktop/apps/identity_verification/routes.tsx

```diff
@@ -0,0 +1,54 @@
+import { buildServerApp } from "reaction/Artsy/Router/server"
+import { buildServerAppContext } from "desktop/lib/buildServerAppContext"
+import { routes } from "reaction/Apps/IdentityVerification/routes"
+import { stitch } from "@artsy/stitch"
+
+export const identityVerification = async (req, res, next) => {
+  if (!res.locals.sd.CURRENT_USER) {
+    return res.redirect(
+      `/login?redirectTo=${encodeURIComponent(req.originalUrl)}`
+    )
+  }
```

Comment:
This can be removed, correct?

Reply:
No, there is no check on the reaction end to catch a no-user-present case, so this will fail with Internal Error/`Error Message: Cannot read property 'identityVerification' of null`. So I am going to leave it here for now.